faunadb/reflect.go

```diff
@@ -23,9 +23,16 @@ func exportedStructFields(aStruct reflect.Value) map[string]reflect.Value {
 			continue
 		}
 
-		fieldName := fieldName(aStructType.Field(i))
+		fieldName, ignore, omitempty, err := parseTag(aStructType.Field(i))
+		if err != nil {
+			//TODO Handle error in case of bad tag options? Currently invalid options are just skipped
```

Comment:
I think we should return an error here since this would be a user mistake that needs correction. Ignoring a malformed tag would be error prone.

Reply:
Done.